pkg/awsutils/awsutils_test.go

```diff
@@ -1316,6 +1316,7 @@ func TestEC2InstanceMetadataCache_buildENITags(t *testing.T) {
 			want: map[string]string{
 				"node.k8s.amazonaws.com/instance_id": "i-xxxxx",
 				"cluster.k8s.amazonaws.com/name":     "awesome-cluster",
+				"eks:eni:owner":                      "amazon-vpc-cni",
```

Comment:
Can we use the variable defined in awsutils? We can then ensure we have tests on the same sources.